var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -577,7 +605,7 @@ def define_enable(suffix, value=None):
             # Thyra is NOT enabled at this point!" leading to eventual build
             # errors if using MueLu because `Xpetra_ENABLE_Thyra` is set to
             # off.
-            options.append(define_trilinos_enable('Thyra', True))
+            # options.append(define_trilinos_enable('Thyra', True))
```

Comment:
Now that thyra is its own variant, can you remove this block and add `conflicts('~thyra', when='+muelu +stratimikos')`?